background.ts

```diff
@@ -842,12 +842,20 @@ ipcMainProxy.handle('service-forward', async(_, service, state) => {
   if (state) {
     const hostPort = service.listenPort ?? 0;
 
-    await k8smanager.kubeBackend.forwardPort(namespace, service.name, service.port, hostPort);
+    await doForwardPort(namespace, service.name, service.port, hostPort);
```

Comment:
⏰

I think, as a follow-up, we can drop this and just have the front end use the API directly. But it probably makes sense to _not_ include that in this PR.

(This also means we can drop the `doForwardPort` and `doCancelForward` functions and just `k8smanager` directly in `BackgroundCommandWorker`; but that would obviously not be done yet.)

Reply:
Ya, I think that may be a larger effort that should be separate from individual feature PRs, but agree that it could be useful architecturally

Reply:
Yes, it can be done later, but one reason to have the API is so that the CLI and GUI go through exactly the same code paths. Ideally the frontend should only communicate with the backend over the API. We are not anywhere close to that, but that should be our the direction.

This is especially important for BATS testing, so we don't have to worry about the GUI doing something slightly different.

Reply:
Filed as #6697